docs/how-to/service-placement.rst

```diff
@@ -1,71 +1,94 @@
 
-The enable command is used as follows:
-  .. code-block:: shell
+Cluster designs that call for additional service instances however can be
+satisfied by manual means.
 
-    $ sudo microceph enable <service> --target <node_hostname> ...
+This is the purpose of the :command:`enable` command. It manually enables a
+new instance of a service on a node.
 
-.. note:: Some services can be provided with additional parameters while enabling them, these can be listed for any service by issuing respective help command.
+The syntax is:
 
-  .. code-block:: shell
+.. code-block:: none
 
-    $ sudo microceph enable <service> --help
+   sudo microceph enable <service> --target <destination> ...
 
-1. Enable radosgw service:
+Where the service value is one of 'mon', 'mds', 'mgr', and 'rgw'. The
+destination is a node name as discerned by the output of the :command:`status`
+command:
 
-  .. code-block:: shell
+.. code-block:: none
 
-    $ sudo microceph status
-    MicroCeph deployment summary:
-    - node1-2c3eb41e-14e8-465d-9877-df36f5d80922 (10.111.153.78)
-      Services: mds, mgr, mon, osd
-      Disks: 3
-    - workbook (192.168.29.152)
-      Services: mds, mgr, mon
-      Disks: 0
-    $ sudo microceph enable rgw --target node1 --port 8080 
-    $ sudo microceph status
-    MicroCeph deployment summary:
-    - node1 (10.111.153.78)
-      Services: mds, mgr, mon, rgw, osd
-      Disks: 3
-    - workbook (192.168.29.152)
-      Services: mds, mgr, mon
-      Disks: 0
+   sudo microceph status
 
-2. Enable mon Service:
+For a given service, the :command:`enable` command may support extra
+parameters. These can be discovered by querying for help for the respective
+service:
 
-  .. code-block:: shell
+.. code-block:: none
 
-    $ sudo microceph enable mon --target <node_hostname> 
+   sudo microceph enable <service> --help
 
-3. Enable mgr Service:
+Example: enable an RGW service
+------------------------------
 
-  .. code-block:: shell
+First check the status of the cluster to get node names and an overview of
+existing services:
 
-    $ sudo microceph enable mgr --target <node_hostname> 
+.. code-block:: none
 
-4. Enable mds Service:
+   sudo microceph status
 
-  .. code-block:: shell
+   MicroCeph deployment summary:
+   - node1-2c3eb41e-14e8-465d-9877-df36f5d80922 (10.111.153.78)
+     Services: mds, mgr, mon, osd
+     Disks: 3
+   - workbook (192.168.29.152)
+     Services: mds, mgr, mon
+     Disks: 0
 
-   $ sudo microceph enable mgr --target <node_hostname> 
+View any possible extra parameters for the RGW service:
 
+.. code-block:: none
+
+   sudo microceph enable rgw --help
+
+To enable the RGW service on node1 and specify a value for extra parameter
+`port`:
+
+.. code-block:: none
+
+   sudo microceph enable rgw --target node1 --port 8080
+
+Finally, view cluster status again and verify expected changes:
+
+.. code-block:: none
+
+   sudo microceph status
+
+   MicroCeph deployment summary:
+   - node1 (10.111.153.78)
```

Comment:
It seems strange to me that first `microceph status` reports a node `node1-2c3eb41e-14e8-465d-9877-df36f5d80922` above, but here it's just `node1`?

Reply:
I was using the existing command output. Maybe we keep it for now and when I get around to actual tool testing I can fill this in with realistic output.